applications/asset_tracker/README.rst

```diff
@@ -52,6 +52,49 @@ The `LTE Link Monitor`_ application, implemented as part of `nRF Connect for Des
 
 By default, the asset tracker supports firmware updates through :ref:`lib_aws_fota`.
 
+.. _power_opt:
+
+Power optimization
+==================
+
+The Asset Tracker can run in three power modes that are configured in the Kconfig file of the application.
+These settings are currently only supported on the nRF9160 DK.
```

Comment:
The modes can be used also on Thingy:91, but they can't be changed runtime using the switches

Reply:
It would be good to mention the configurations used to enable eDRX and PSM as well.

Reply:
@jtguggedal Added the sentence: Although these power modes are supported in Thingy:91, the settings cannot be changed at runtime.

@tokangas  The configurations are mentioned in the Power Optimization section.

Reply:
I think this is a bit confusing, because it first says that these settings are only supported on the DK and then continues saying that the modes are supported in Thingy:91, but can't be changed at runtime.

The problem is that there are two different things here:

- changing between eDRX and PSM during runtime (supported only by DK)
- build time configuration for enabling eDRX and/or PSM (supported by both)

The first one can be enabled by setting CONFIG_POWER_OPTIMIZATION_ENABLE and then the used power power saving mode (eDRX/PSM) can be selected using the HW switch as described by the text currently.

However, to enable eDRX and/or PSM for Thingy:91 (this of course also works with DK as well), different configurations are used. eDRX can be enabled by setting CONFIG_LTE_EDRX_REQ and for PSM there's CONFIG_GPS_CONTROL_PSM_ENABLE_ON_START, which enables PSM when the GPS starts. CONFIG_POWER_OPTIMIZATION_ENABLE does nothing on a Thingy:91.

I think we should either change this to clearly describe the runtime power optimization mode change and say that it's only supported by DK or cover both possibilities separately.

Reply:
Further comment to earlier chapter:
GPS coordinates | GPS | NEMA Gxxx string ---> **NMEA** Gxxx string

Reply:
Added the suggested information.